source/_integrations/sighthound.markdown

```diff
@@ -63,3 +63,47 @@ source:
       required: false
       type: string
 {% endconfiguration %}
+
+To verify the integration, check if a new entity is appeared as `image_processing.sighthound_my_cam`
+
+## Process an Image
+
+When you want to process an image, you have to call `image_processing.scan` service and listen to the sighthound.person_detected and/or sighthound.vehicle_detected events.
+
+A simple example is via 2 two automations:
+
+- The first automation is triggered, when a motion is detected. It calls the `image_processing.scan` service to send the camera image to the sighthound server for processing.
+
+- The second automation is triggered by a `sighthound.vehicle_detected` event. It sends a notification to a phone.
+
+```yaml
+# Example automations.yaml entry
+- id: '...'
```

Comment:
Please format this according to our documentation and YAML standards.
For more information, check out developer portal

Reply:
Thank you for the review @frenck. I didn't know about the Home Assistant developer portal. I have read the documentation standards and the YAML Style Guide. I hope that the updated sample confirms with them.

Funnily, my automations.yaml, generated by Home Assistant, uses single quotes for strings.